experimental/Schemes/BlowupMorphism.jl

```diff
@@ -617,20 +617,3 @@ function pushforward(f::BlowupMorphism, g::VarietyFunctionFieldElem)
   pfg = fraction(pullback(phi)(OO(V)(numerator(h))))//fraction(pullback(phi)(OO(V)(denominator(h))))
   return FY.(pfg)
 end
-
-@attr AbsSpecMor function isomorphism_on_open_subset(phi::AbsCoveredSchemeMorphism)
```

Comment:
Why is this and the following removed? It is called later on in line 496

Reply:
Good catch! Looks like a merge gone wrong.